### PR TITLE
GUIAnalysis

### PR DESCRIPTION
**Pull request by egolol**
first commmit, works until tab

### CODE REVIEW

File: braph2/workflows/Structural/AnalysisST_BUD.m [hunk 1]
```diff
@@ -29,13 +29,16 @@
     %  getComparisonClass           - returns the class of the comparison
     %  getAvailbleSettings          - returns the available settings
     %
+    % AnalysisST_BUD Plot UIPanels
+    %  getMainPanelMeasurePlot      - creates a uipanel for a plot
```
[the remark this answers is not in the document]
Comment (egolol) [reply braph2/workflows/Structural/AnalysisST_BUD.m:33]: I will put the nodal and binodal in another branch

File: braph2/analysis/Comparison.m [hunk 2]
```diff
@@ -42,6 +42,10 @@
     %  getAvailableSettings     - returns available settings to the comparison
     %  getRandomComparion       - returns a new comparison
     %
+    % Comparison Plot methods (Static)
+    %  getComparisonPanel       - creates a UI Figure for Comparison Settings
+    %  getChildPanel            - returns a dynamic UIPanel
```
Comment (giovannivolpe) [inline braph2/analysis/Comparison.m:47]: remove

File: braph2/analysis/Comparison.m [hunk 3]
```diff
@@ -377,6 +381,393 @@ function setGroups(c, group_1, group_2)
             sub = eval([comparison_class '(id, label, notes, atlases, measure_code, group_1, group_2, varargin{:})']);
         end
     end
+    methods (Static)  % Plot general panel
+        function GUIComparison(comparison_class, analysis)
```
Comment (giovannivolpe) [inline braph2/analysis/Comparison.m:385]: Move it as a separate function in separate file

File: braph2/analysis/Measurement.m [hunk 4]
```diff
@@ -42,6 +42,10 @@
     %  getAvailableSettings     - returns available settings to the measurement
     %  getMeasurement           - returns a new measurement
     %
+    % Measurement Plot Methods (Static)
+    %  getMesurementPanel       - creates a UIFigure for Measurement Settings
+    %  getChildPanel            - returns a dynamic UIPanel
```
Comment (giovannivolpe) [inline braph2/analysis/Measurement.m:47]: remove method childPanel

File: braph2/analysis/Measurement.m [hunk 5]
```diff
@@ -370,19 +374,426 @@ function setGroup(m, group)
             sub = eval([measurement_class  '(id, label, notes, atlases, measure_code, group, varargin{:})']);
         end
     end
+    methods (Static)  % Plot general panel
+        function GUIMeasurement(measurement_class, analysis)
```
Comment (giovannivolpe) [inline braph2/analysis/Measurement.m:378]: separate file

File: braph2/analysis/RandomComparison.m [hunk 6]
```diff
@@ -42,6 +42,10 @@
     %  getAvailableSettings     - returns available settings to the random comparison
     %  getRandomComparion       - returns a new random comparison
     %
+    % RandomComparison Plot methods (Static)
+    %  getRandomComparisonPanel - creates a UI Figure for RandomComparison Settings
+    %  getChildPanel            - returns a dynamic UIPanel
```
Comment (giovannivolpe) [inline braph2/analysis/RandomComparison.m:47]: Rrmove

File: braph2/analysis/RandomComparison.m [hunk 7]
```diff
@@ -371,6 +375,386 @@ function setGroup(rc, group)
             sub = eval([randomComparisonClass '(id, label, notes, atlas, measure_code, group, varargin{:})']);
         end
     end
+    methods (Static)  % Plot general panel
+        function GUIRandomComparison(comparison_class, analysis)
```
Comment (giovannivolpe) [inline braph2/analysis/RandomComparison.m:379]: Separate file

**Review by giovannivolpe — CHANGES_REQUESTED** [review]

**Pull request merged**